app/assets/javascripts/application.js

```diff
@@ -13,3 +13,6 @@
 //= require jquery
 //= require jquery_ujs
 //= require_tree .
+//= require appointment.js
+//= require patient.js
+//= require bootstrap.min
```

Comment:
If you have require_tree you don't need to explicitly require appointment and patient. Also, require bootstrap above your own js but below jquery.